src/mlpack/methods/cf/cf_main.cpp

```diff
@@ -71,6 +80,18 @@ PROGRAM_INFO("Collaborative Filtering", "This program performs collaborative "
     " - 'SVDCompleteIncremental' -- SVD complete incremental learning\n"
     " - 'BiasSVD' -- Bias SVD using a SGD optimizer\n"
     " - 'SVDPP' -- SVD++ using a SGD optimizer\n"
+    "\n\n"
+    "The following neighbor search algorithms can be specified via" +
+    " the " + PRINT_PARAM_STRING("neighbor_search") + " parameter:"
```

Comment:
Picky comment, we could add a new line here for readability.
```suggestion
    " the " + PRINT_PARAM_STRING("neighbor_search") + " parameter:"
    "\n"
```